pipelines/tasks/get-package-versions-list.yaml

```
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-package-versions-list
+spec:
+  volumes:
+    - name: unpinned-packages
+      emptyDir:
+        medium: memory
```

`Memory` must be [capitalized](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir):

```suggestion
        medium: Memory
```

otherwise you get:

```
0s          Warning   FailedMount               pod/test-pkglist-pr-resolve-dependencies-pod           MountVolume.SetUp failed for volume "unpinned-packages" : unknown storage medium "memory"
```